src/types.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
```

Comment:
According to [this video](https://www.loom.com/share/eba664a893454540a3f7cfb77a96540c), Tthere's no need to specify types of those properties you didn't use in your code